tdrs-backend/docs/super_user_management.md

```diff
@@ -0,0 +1,62 @@
+# Superuser Creation
```

Comment:
Documents how to change roles using the CLI, specifically the built in `superuser` role